packages/sdk-middleware-http/test/http.spec.js

```diff
@@ -134,6 +166,38 @@ describe('Http', () => {
       httpMiddleware(next)(request, response)
     }))
 
+  test('execute a get request with getAbortController short timeout (fail)', () =>
+    new Promise((resolve, reject) => {
+      const request = createTestRequest({
+        uri: '/foo/bar',
+      })
+      const response = { resolve, reject }
+      const next = (req, res) => {
+        expect(res).toEqual({
```

Comment:
With these "delayed" `expect` blocks it could be useful to add a `expect.assertions(1)` on L170 to ensure that the expectation is actually called and we don't silently pass the test.

Reply:
Added the expect.assertions